addons/skin.estuary/xml/Custom_1109_TopBarOverlay.xml

```diff
@@ -148,7 +148,7 @@
 		<control type="group">
 			<animation effect="slide" end="0,-90" time="300" tween="sine" easing="inout" condition="$EXP[infodialog_active]">conditional</animation>
 			<animation effect="slide" start="0,-200" end="0,0" time="300" tween="cubic" easing="out">VisibleChange</animation>
-			<visible>Player.ShowInfo | Window.IsActive(fullscreeninfo) | Player.ShowTime | Window.IsActive(videoosd) | Window.IsActive(musicosd) | Window.IsActive(playerprocessinfo) | !String.IsEmpty(Player.SeekNumeric) | !String.IsEmpty(PVR.ChannelNumberInput) | Window.IsActive(pvrosdchannels) | Window.IsActive(pvrchannelguide) | ![Player.Seeking | Player.DisplayAfterSeek | Player.Forwarding | Player.Rewinding | Player.Paused + !Window.IsActive(visualisation)]</visible>
+			<visible>Player.ShowInfo | Window.IsActive(fullscreeninfo) | Player.ShowTime | Window.IsActive(videoosd) | Window.IsActive(musicosd) | Window.IsActive(playerprocessinfo) | !String.IsEmpty(PVR.ChannelNumberInput) | Window.IsActive(pvrosdchannels) | Window.IsActive(pvrchannelguide) + ![Player.Seeking | Player.DisplayAfterSeek | Player.Forwarding | Player.Rewinding | Player.Paused + !Window.IsActive(visualisation)]</visible>
```

Comment:
i know these conditions are a nightmare but, changing an OR to an AND can't be right ?
the top bar is now missing during music playback...

before:
![before](https://user-images.githubusercontent.com/687265/97206062-8b189f00-17b8-11eb-9c1e-253918316e63.jpg)


after:
![after](https://user-images.githubusercontent.com/687265/97206080-8fdd5300-17b8-11eb-858e-ac3bc82833df.jpg)

Reply:
Hmm. I did check music and it seemed ok, will recheck.

Reply:
Yeah it's working ok here, will double check to make sure I have push the wrong changes.

Reply:
Ah there's nothing at the top only when paused, I did indeed miss that as I only tried seeking actions in music viz.

Reply:
@ronie can you try again please, pushed a change that looks to have fixed it and did a quick test with video to ensure no regressions but only had time for a quick test.